lib/loaders/loaders_with_authentication/impulse.js

```diff
@@ -30,5 +32,13 @@ export default (accessToken, userID) => {
       },
       { method: "POST" }
     ),
+    markMessageReadLoader: impulseLoader(
```

Comment:
In our `event` framework for Impulse/Radiation, marking a message as read means logging an 'open' event for the relevant delivery.